### PR TITLE
Integrated ungroup clusters api

### PR DESCRIPTION
**Pull request by ssvaidehi**

### CODE REVIEW

**Review by steven-collins-omega — CHANGES_REQUESTED** [review]
I'd really like to see something added to https://github.com/IBM/satcon-client-go/blob/main/test/integration/groups_test.go to prove this really works. Don't feel like you need to follow the pattern of what's there; in fact the integration tests are pretty monolithic which I'm not a fan of. I'd prefer a test that just tests this, doing minimal setup and teardown.

File: test/integration/groups_test.go [hunk 1]
```diff
@@ -166,6 +166,10 @@ var _ = Describe("Groups", func() {
 			Expect(group1.Clusters[0].ClusterID).To(Equal(newClusterDetails.ClusterID))
 			Expect(group1.Clusters[0].Name).To(Equal(clusterName))
 
+			//Remove cluster from group
+			_, err = c.Groups.UnGroupClusters(testConfig.OrgID, newGroupDetails.UUID, []string{newClusterDetails.ClusterID})
+			Expect(err).NotTo(HaveOccurred())
+
```
Comment (steven-collins-omega) [inline test/integration/groups_test.go:172]: Just testing that it doesn't error out isn't enough. I suggest the following (at each step `Expect`ing there is no error, of course):
- Create (register) another couple of clusters
- Add those to the group
- Use your new method to remove one of the clusters
- List the clusters in the group
- Verify that the cluster you just removed is no longer there, but the other clusters still are

**Review by steven-collins-omega — CHANGES_REQUESTED** [review]
Requested changes. The test needs to test functionality, not just that it doesn't throw an error

File: test/integration/groups_test.go (hunk 1 above)
Comment (ssvaidehi) [reply test/integration/groups_test.go:172]: Added Expects to make sure cluster was getting removed from group

**Review by steven-collins-omega — APPROVED** [review]

**Pull request merged**